Sources/StreamChatUI/CommonViews/Attachments/AttachmentsPreviewVC.swift

```diff
@@ -115,25 +120,6 @@ open class AttachmentsPreviewVC: _ViewController, ComponentsProvider {
     }
     
     open func setupVerticalStackView() {
-        // Disable scroll when not needed
-        scrollView.isScrollEnabled = content.count > maxNumberOfVerticalItems
-
-        // If the content is bigger than the max vertical items and the scroll view height
-        // constraint is not yet created, append to the vertical constraint and activate it.
-        if content.count > maxNumberOfVerticalItems {
-            if scrollViewHeightConstraint == nil {
-                scrollViewHeightConstraint = scrollView.heightAnchor.pin(
-                    lessThanOrEqualToConstant: scrollView.frame.size.height
-                )
-                scrollViewHeightConstraint?.isActive = true
```

Comment:
If we are not using `scrollViewHeightConstraint` anymore we should remove this property. Otherwise, this is dead code. If we are not removing it just because of breaking changes, it doesn't make sense because it is still a breaking change, but a silent one, which is worse than having a normal breaking change.